tests/integration/test_cli.py

```diff
@@ -201,6 +201,9 @@ def test_no_paginate_arg(self):
         p = aws('s3api list-objects --bucket %s --no-paginate' % bucket_name)
         self.assertEqual(p.rc, 0, p.stdout + p.stderr)
 
+        p = aws('s3api list-objects --bucket %s' % bucket_name)
```

Comment:
Is this just to make sure pagination works? Its under the no pagination test though.